cw20/base/schema/query_msg.json

```diff
@@ -3,162 +3,12 @@
   "title": "QueryMsg",
   "anyOf": [
     {
-      "description": "Returns the current balance of the given address, 0 if unset. Return type: BalanceResponse.",
       "type": "object",
       "required": [
-        "balance"
+        "get_count"
```

Comment:
can you check if this change impacts anything in the `docs`?
or the dapp-examples repo

Reply:
Issue here is that we cannot update schema for contract so will copy the ones of cw20_base for the time being
```  invalid character `{` in package name: `{{project-name}}`, the first character must be a Unicode XID start character (most letters or `_`)``` Since it is a template

Reply:
or possibly would be best to remove schema completely as it is only a template